web_monitor/settings.py

```diff
@@ -128,3 +128,9 @@
 
 # AUTH
 LOGIN_URL = '/account/login/'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'devinitautomailer@gmail.com'
```

Comment:
@davidebukali or @edwinmp ... configure these to prioritise configs from the `.env` file.

Reply:
@edwinmp updated these email configurations to use `os.environ.get`